lib/net/validations.rb

```diff
@@ -9,6 +9,8 @@ module Validations
     HOST_REGEXP ||= /\A(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\z/
     MASK_REGEXP ||= /\A((255.){3}(0|128|192|224|240|248|252|254))|((255.){2}(0|128|192|224|240|248|252|254).0)|(255.(0|128|192|224|240|248|252|254)(.0){2})|((128|192|224|240|248|252|254)(.0){3})\z/
 
+    HOST_REGEXP_ERR_MSG = "hostname must start with and contain alphanumerical characters and can contain dashes and dots"
```

Comment:
Please extract the string for translation here using `N_()`. Also consider perhaps rephrasing to something a bit more concise such as "hostname may only contain letters, numbers, dashes and dots".